packages/e2e/src/page-objects/modal.backup.restore.ts

```diff
@@ -0,0 +1,64 @@
+import { ChainablePromiseElement } from "webdriverio"
```

Comment:
modal.backup.restore.ts -> modal-backup-restore.ts 
modal.backup.restore.ts -> modal-backup-restore.page.ts (Maybe page suffix if you needed 🤔)

why the file name is divided by dots? 🤔 I recommended splitting words via dash like the rest component of the project.

If you would like more information about the file naming convention  please review this document 👉 https://appnroll.atlassian.net/wiki/spaces/CP/pages/1682571271/Files+naming+convention